lib/web_ui/flutter_js/src/loader.js

```diff
@@ -87,11 +88,25 @@ export class FlutterLoader {
       }
     }
 
+    /**
+     * @param {import("./types").ApplicationBuild} build
+     * @param {import("./types").WebRenderer} renderer
+     **/
+    const buildContainsRenderer = (build, renderer) => {
+      switch (build.renderer) {
+        // The "auto" build contains both canvaskit and html renderers.
```

Comment:
Is this true? does the build configuration when building in "auto" contain info for both html and canvaskit?

Reply:
Yes, it contains both renderers. How else would it select different renderers at runtime based on mobile web vs desktop web?

Reply:
This is what I see:

```
_flutter.buildConfig = {"engineRevision":"b2d93a64cbc7dbd98ddac5d7ec5aef9e814ac851","builds":[{"compileTarget":"dart2js","renderer":"auto","mainJsPath":"main.dart.js"}]};
```

I was expecting it to contain entries for both "canvaskit" and "html", and not an entry called "auto", sorry!